src/cluster/src/start/local.rs

```diff
@@ -510,4 +530,48 @@ impl LocalClusterInstaller {
             .map_err(|_| LocalInstallError::Other("SPU server failed to start".to_string()))?;
         Ok(())
     }
+
+    /// Check to ensure SPUs are all running
+    async fn confirm_spu(&self, spu: u16) -> Result<(), LocalInstallError> {
+        use std::env;
+        use fluvio::Fluvio;
+
+        let delay: u64 = env::var("FLV_SPU_DELAY")
+            .unwrap_or_else(|_| "1".to_string())
+            .parse()
+            .unwrap_or(1);
+
+        println!("waiting for spu to be provisioned for: {} seconds", delay);
+
+        sleep(Duration::from_secs(delay)).await;
+
+        let client = Fluvio::connect().await.expect("sc ");
+        let mut admin = client.admin().await;
+
+        // wait for list of spu
+        for _ in 0..30u16 {
+            let spus = admin.list::<SpuSpec, _>(vec![]).await.expect("no spu list");
+            let live_spus = spus
+                .iter()
+                .filter(|spu| {
+                    spu.status.is_online() && !spu.spec.public_endpoint.ingress.is_empty()
```

Comment:
ingress checking is not needed for local cluster

Reply:
This is code that was already in the project, I just moved it here from `src/cluster/src/cli`. What would need to change in order for this check to be correct?

Reply:
Should have flag to check for ingress or not.   For local, `false`, for k8, `true`